.script/tests/detectionTemplateSchemaValidation/DetectionTemplateSchemaValidationTests.cs

```diff
@@ -10,19 +12,50 @@
 
 namespace Kqlvalidations.Tests
 {
-    public class DetectionTemplateStructureValidationTests
+    public class DetectionTemplateSchemaValidationTests
     {
         private static readonly string DetectionPath = DetectionsYamlFilesTestData.GetDetectionPath();
 
         [Theory]
         [ClassData(typeof(DetectionsYamlFilesTestData))]
         public void Validate_DetectionTemplates_HaveValidTemplateStructure(string detectionsYamlFileName)
```

Comment:
nit: HaveValidTemplate**Schema**?

Reply:
The first one is about the structure, the second is about the connectors IDS.
The overall name change (structure -> schema) is because of the addition of the new test, so it's more general one.